features/support/sphinx.rb

```diff
@@ -1,23 +1,20 @@
 require 'thinking_sphinx'
 require 'thinking_sphinx/test'
 
-Before "not @search" do
-  ::ThinkingSphinx::Test.disable_search_jobs!
-end
-
 Before('@search') do
   Sidekiq::Job.clear_all
   ::ThinkingSphinx::Test.stop
   ::ThinkingSphinx::Test.clear
   ::ThinkingSphinx::Test.init
   $searchd_autostop_installed ||= ::ThinkingSphinx::Test.autostop
   ::ThinkingSphinx::Test.wait_start
+
+  ::ThinkingSphinx::Test.enable_search_jobs!
 end
 
 After '@search' do
+  ::ThinkingSphinx::Test.disable_search_jobs!
   ::ThinkingSphinx::Test.stop
 end
 
-After "not @search" do
-  ::ThinkingSphinx::Test.enable_search_jobs!
-end
+::ThinkingSphinx::Test.disable_search_jobs!
```

Comment:
```suggestion
BeforeAll do
  ::ThinkingSphinx::Test.disable_search_jobs!
end
```

I actually have no idea if this works without wrapping it in `BeforeAll` (or `Before`), but to me it seems to be more correct to add a wrapper https://cucumber.io/docs/cucumber/api/?lang=ruby

Reply:
But it seems to be only available in some newer cucumber version. So we need to stay like this for now. Cucumber updates are more often than not very involved.

```
undefined method `BeforeAll' for main:Object
Did you mean?  Before (NoMethodError)
```

Reply:
Seems that upgrading to cucumber 7.1 doesn't break anything https://github.com/3scale/porta/pull/3806

`BeforeAll` should be available there: https://github.com/cucumber/cucumber-ruby/blob/main/CHANGELOG.old.md#710